game/ffmpeg.hh

```diff
@@ -184,6 +123,8 @@ class FFmpeg {
 	bool terminating() { return m_quit_future.wait_for(0s) == std::future_status::ready; }
 
 	class eof_error: public std::exception {};
+	/// Returns filename; only used for debugging.
+	fs::path const& getFilename() const { return m_filename; }
```

Comment:
If its only used for debugging you can either create a log line on debug mode, else just remove this code

Reply:
The log line _is_ there, I just commented the actual line out so as to not pollute the log any more.